source/includes/library-reference/_brain.html.md

```diff
@@ -0,0 +1,47 @@
+# Brain Class
+
+Used to manage a BRAIN instance and talk with the server backend. Contains
+state information about the BRAIN and can be used to upload/download inkling
+to and from the BRAIN on the server.
+
+Requires a shared pointer to a `bonsai::Config` and a BRAIN name.
+
+## Brain(config, name)
+
+Construct a BRAIN using an existing configuration and BRAIN name.
+
+- `config` Shared pointer to previously created `bonsai::Config`.
+- `name`   BRAIN name as specified on the server. If name is empty the
```

Comment:
empty vs None... usually Python favors None with a default param name=None where in the docs you would say something like `name defaults to the BRAIN name in config when name is not specified` or something like that.

Reply:
1. this is not a docstring
2. this is C++ documentation first, python documentation second.
3. it requires an empty string or null string.